shared/health/src/test/java/io/pravega/shared/health/impl/ContributorRegistryTests.java

```diff
@@ -0,0 +1,378 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.shared.health.impl;
+
+import io.pravega.shared.health.ContributorRegistry;
+import io.pravega.shared.health.HealthContributor;
+import io.pravega.shared.health.TestHealthIndicators.SampleHealthyIndicator;
+import io.pravega.shared.health.TestHealthIndicators.SampleFailingIndicator;
+
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Random;
+import java.util.Set;
+
+@Slf4j
+public class ContributorRegistryTests {
+
+    private static final int BOUND = 100;
+
+    ContributorRegistry registry;
+
+    @Before
+    public void before() {
+        registry = new ContributorRegistryImpl();
+    }
+
+
+    @After
+    public void after() {
+        registry.clear();
```

Comment:
close?

Reply:
There are no resources to close from a `ContributorRegistry`.